mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -834,4 +722,105 @@ class PopulateParams : public PopulateParamsBase {
                               int64_t &gemmCDstPerWrite, int64_t &gridSize);
 };
 
+class PopulateParamsXDL : public PopulateParamsBase {
+private:
+  llvm::SmallVector<InitParamsXDL, 4> initParameters = {
+      // M/block N/block K/block M/wave N/wave
+      {128, 128, 16, 64, 64},
```

Comment:
@jerryyin could you check with @zjing14 to ensure these initial vector looks correct?

Reply:
Will add him in the next review.